app/code/Magento/InventoryApi/Test/Api/SourceItemsSave/ValidationTest.php

```diff
@@ -227,6 +236,15 @@ public function failedValidationDataProvider(): array
                                 'field' => SourceItemInterface::QUANTITY,
                             ],
                         ],
+                        [
+                            'message' => 'Product cannot have "%status" "%in_stock" while ' .
```

Comment:
We don't need to change message if we have situation `'"%field" should be numeric.'`

Reply:
Fixed

Reply:
Done https://github.com/magento-engcom/msi/issues/449